airflow/models/dag.py

```diff
@@ -1964,7 +1964,13 @@ def partial_subset(
             tasks, in addition to matched tasks.
         :param include_upstream: Include all upstream tasks of matched tasks,
             in addition to matched tasks.
+        :param include_direct_upstream: Include all tasks directly upstream of matched
+            and downstream (if include_downstream = True) tasks. Can not be use when
+            include_upstream = True - would be a noop with overhead.
```

Comment:
Before it was "Include all tasks directly upstream of matched", now it has the "and downstream" part.